Frontend/src/data/customer/columns.js

```diff
@@ -93,6 +93,22 @@ export default [
   {
     title: "Telefonnummer",
     key: "telephone_number",
+    display: (value) => {
+      if (!value) return value;
+      let formattedValue = value.replace(/ /g, "");
+      if (formattedValue.startsWith("+")) {
+        formattedValue = formattedValue.replace(
+          /(\d{5})(\d{4})(\d{1,})/,
+          "$1 $2 $3"
+        );
+      } else {
+        formattedValue = formattedValue.replace(
+          /(\d{4})(\d{4})(\d{1,})/,
+          "$1 $2 $3"
```

Comment:
this would yield `+XXXX XXXX REST`, is that correct? 

looks good then! :)

Reply:
If a number starts with "+" it returns `+xxxxx xxxx rest`
If a number does not start with "+" it returns `xxxx xxxx rest`

Is this the way it should be?

Reply:
Yes, perfect, that's exactly what I meant :) was just asking is I understand the code correctly